lib/modal/modal.scss

```diff
@@ -41,7 +41,7 @@
   pointer-events: none;
 }
 
-.#{$pre}-modal.#{$pre}-mounted {
+.#{$pre}-modal.#{$pre}-modal--mounted {
```

Comment:
If we're following the same updated CSS flag naming convention, I woulda thought this would need to be: `.#{$pre}-modal.mounted`... Unless I have misunderstood. Don't want to introduce something new, if we don't  need to.